apps/web/lib/posthog.ts

```diff
@@ -0,0 +1,10 @@
+import { PostHog } from "posthog-node";
+
+export default function PostHogClient() {
+  const posthogClient = new PostHog(process.env.NEXT_PUBLIC_POSTHOG_KEY!, {
+    host: process.env.NEXT_PUBLIC_POSTHOG_HOST,
```

Comment:
do they not have a default value?

Reply:
We could set the API but the key is not a default values.